packages/cli/src/active-executions.ts

```diff
@@ -229,7 +242,23 @@ export class ActiveExecutions {
 		}
 	}
 
-	getExecution(executionId: string): IExecutingWorkflowData {
+	private groupExecutionIds() {
```

Comment:
Can we add a JSDoc comment about the purpose of this method and what it returns

Reply:
I can do that. but TBH, I don't like method name. Any suggestions for a better name? The purpose of the method is to partition executions into 
1. ones that need to be cancelled because they have a pending response-promise, which can't be retained across restarts
2. all other active executions